lms/envs/common.py

```diff
@@ -1230,10 +1230,26 @@
 )
 
 proctoring_js = (
-    ['proctoring/js/models/*.js'] +
-    ['proctoring/js/collections/*.js'] +
-    ['proctoring/js/views/*.js'] +
-    ['proctoring/js/*.js']
+    [
+        'proctoring/js/models/proctored_exam_allowance_model.js',
+        'proctoring/js/models/proctored_exam_attempt_model.js',
+        'proctoring/js/models/proctored_exam_model.js'
+    ] +
+    [
+        'proctoring/js/collections/proctored_exam_allowance_collection.js',
+        'proctoring/js/collections/proctored_exam_attempt_collection.js',
+        'proctoring/js/collections/proctored_exam_collection.js'
+    ] +
+    [
+        'proctoring/js/views/Backbone.ModalDialog.js',
+        'proctoring/js/views/proctored_exam_add_allowance_view.js',
+        'proctoring/js/views/proctored_exam_allowance_view.js',
+        'proctoring/js/views/proctored_exam_attempt_view.js',
+        'proctoring/js/views/proctored_exam_view.js'
+    ] +
+    [
+        'proctoring/js/proctored_app.js'
+    ]
```

Comment:
Note to reviewers, we had to expand out the actual JS file because the wildcard stopped working in devstack environments due to a recent change to the JS pipeline. AWS environments however worked.